fake_fire/init.lua

```diff
@@ -172,6 +172,11 @@ local sbox = {
 	fixed = { -8/16, -8/16, -8/16, 8/16, -6/16, 8/16},
 }
 
+local wtex = "default_junglewood.png"
+if not minetest.get_modpath("default") then
+	wtex = "[combine:16x16^[noalpha^[colorize:#563d2d"
+end
+
```

Comment:
negated default conditions are more difficult to understand; better to write something like
```lua
local wtex = "[combine:16x16^[noalpha^[colorize:#563d2d"
if minetest.get_modpath("default") then
	wtex = "default_junglewood.png"
end

```

Reply:
might replace the handful of texture instances with something like materials.lua

Reply:
https://github.com/mt-mods/homedecor_modpack/pull/28/commits/6370ebe1c890a69514fec45f43222fd12d4e5b82 fixed